### DISCUSSION TITLE
Run.sh already exists when running workflow

### DISCUSSION THREAD

**Discussion by philanyware** [community/community#25263] [Actions]
<p>Hi,<br>
I created a self hosted runner on my server (Red Hat 4.8.5-44) and it runs the workflow but fails with this error:</p>
<p>[2022-01-30 20:13:51Z ERR JobRunner] System.IO.IOException: The file ‘/home/searchdev/actions-runner/run.sh’ already exists.<br>
at System.IO.FileSystem.CreateDirectory(String fullPath)<br>
at System.IO.Directory.CreateDirectory(String path)<br>
at GitHub.Runner.Worker.JobRunner.RunAsync(AgentJobRequestMessage message, CancellationToken jobRequestCancellationToken)</p>
<p>Why would it be trying to recreate the run.sh file and how can I fix this error please?</p>
<p>Note that I installed the runner by copying and pasting the commands from the <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> Add New Runner page for linux and X64:</p>
<pre><code class="lang-auto">### Download
# Create a folder
$ mkdir actions-runner &amp;&amp; cd actions-runner
# Download the latest runner package
$ curl -o actions-runner-linux-x64-2.286.1.tar.gz -L https://github.com/actions/runner/releases/download/v2.286.1/actions-runner-linux-x64-2.286.1.tar.gz 
# Optional: Validate the hash
$ echo "7b1509c353ea4e6561b2ed2e916dcbf2a0d8ce21172edd9f8c846a20b6462cd6 actions-runner-linux-x64-2.286.1.tar.gz" | shasum -a 256 -c 
# Extract the installer
$ tar xzf ./actions-runner-linux-x64-2.286.1.tar.gz

### Configure
# Create the runner and start the configuration experience
$ ./config.sh --url https://github.com/anywarewebmarketing/paradigm-pcs-react-search.pp.co.nz --token ACQNKP2MCQQPTEWFD7TTQB3B63U7S 
# Last step, run it!
$ ./run.sh
</code></pre>
<p>Thanks,<br>
Phil</p>

**Comment by songwookim** [comment]
<p>i got same message !<br>
how about to delete the folder and reinstall ?</p>

**Comment by philanyware** [comment, marked as answer]
<p>For me it was a disk space problem. I deleted the runner and created it again.<br>
Then I had resource problems - the runner uses a lot of CPU, RAM and IO. I also had to allocate more resources to the hosting account. After that it runs ok.</p>
